utils_linux.go

```diff
@@ -81,6 +81,8 @@ func newProcess(p specs.Process) (*libcontainer.Process, error) {
 		Label:           p.SelinuxLabel,
 		NoNewPrivileges: &p.NoNewPrivileges,
 		AppArmorProfile: p.ApparmorProfile,
+		ConsoleWidth:    uint16(p.ConsoleSize.Width),
+		ConsoleHeight:   uint16(p.ConsoleSize.Height),
```

Comment:
Yeh this seems reasonable. Have updated the PR - let me know if you think this is an appropriate solution. Thanks.